### DISCUSSION TITLE
'.' as entity after fine-tuning NER BERT

### DISCUSSION THREAD

**Discussion by DaniXavier** [explosion/spaCy#11564] [Help: Other Questions]
Hello. I followed this [post](https://dzone.com/articles/how-to-fine-tune-bert-transformer-with-spacy-3) to train my own NER BERT model. The goal is to be able to recognize entities like BRAND and STYLE in different texts. I have a training set with more than 6k entries. When I evaluate a text like: "Vera Wayne makes beautiful wedding dresses."
I get the following:
[('Vera Wayne', 'BRAND'), ('dresses', 'PRODUCT'), ('.', 'BRAND')]

The first two entities are correct but the last one is not and that is happening with all my sentences.
Why am I getting all periods as BRAND?
I already checked my training data and all the tokens containing just a period are classified as O (I'm using IOB format).
However I have BRANDS which have period on their names, like 'Mi.iM'.
What do you think is causing this issue? How can I solve it?

**Comment by polm** [comment]
That sounds pretty weird. If you don't have any examples of a period being annotated as a BRAND I'm not really sure what could cause that. Could you give a few examples of your training data to help us better understand your task? Also, try using `spacy debug data` if you haven't done so yet.
